VK_App/VK_App/Controller/FriendsListTableViewController.swift

```diff
@@ -124,14 +126,19 @@ final class FriendsListTableViewController: UITableViewController {
     }
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        guard let people = sectionsMap[sectionsTitles[indexPath.section]]?[indexPath.row]
+        guard
+            let people = sectionsMap[sectionsTitles[indexPath.section]]?[indexPath.row]
         else { return UITableViewCell() }
-        guard let cell = tableView.dequeueReusableCell(
-            withIdentifier: Constants.cellIdentifier,
-            for: indexPath
-        ) as? FriendsListViewCell
+        guard
+            let cell = tableView.dequeueReusableCell(
+                withIdentifier: Constants.cellIdentifier,
+                for: indexPath
+            ) as? FriendsListViewCell
         else { return UITableViewCell() }
-        cell.configure(people)
+        cell.configure(
+            people,
+            image: photoCacheService?.photo(atIndexPath: indexPath, byUrl: people.userPhoto ?? "") ?? UIImage()
```

Comment:
```suggestion
            image: photoCacheService?.photo(atIndexPath: indexPath, byUrl: people.userPhoto ?? ) ?? UIImage()
```
Константы.
 Если инициализируешь через lazy не нужен будет ?? UIImage/ и такая запись "?? UIImage()" мне лично никогда не нравилась, стараюсь избегать. Дефолтное значение лучше 0 или "" а если дефолтом прокидывать класс мне кажется это не лучшая практика. Можешь в методе cell принимать опционал, так тоже избавишься от этого UIImage()